Automattic/wp-calypso
# Migrate post editor styles to webpack

#### Changes proposed in this Pull Request

* Migrate all styles in `client/post-editor` to our webpack CSS pipeline

#### Testing instructions

* Open up the post editor and try to use every available piece of UI
* Does it look like on master? Any visual regressions?
* Also check the media modal in site-settings

related #27515
